components/MashCards/MainCard.tsx

```diff
@@ -0,0 +1,80 @@
+import { MashReferral } from 'types';
+import ContactCard from './ContactCard';
+import InitialDecisionCard from './InitialDecisionCard';
+import ScreeningCard from './ScreeningCard';
+import FinalDecisionCard from './FinalDecisionCard';
+
+interface Props {
+  filter: string;
+  mashReferrals: MashReferral | any;
```

Comment:
why the | any?

Reply:
ah I know why, it should be MashReferral[] not MashReferral.

That will mean we are stating the type of mashReferrals is an array of type MashReferral.
It needs to be an array so we can loop over it (the .map call inside the JSX).
I expect the | any got rid of this error but in general we want to avoid | any. Would be easier to explain on a call if you want an explanation for why we want to avoid any.